src/components/Col/Col.tsx

```diff
@@ -27,7 +28,23 @@ export const colSizes = [
   "10",
   "11",
   "12",
-];
+] as const;
+
+export type ColSize = typeof colSizes[number];
+
+type Props = {
+  size: ColSize;
+  children?: ReactNode;
+  className?: string;
+  element?: ElementType;
+  emptyLarge?: ColSize;
+  emptyMedium?: ColSize;
+  emptySmall?: ColSize;
+  large?: ColSize;
+  medium?: ColSize;
+  small?: ColSize;
+  [x: string]: ReactNode;
```

Comment:
Is `ReactNode` right here? Shouldn't it be a dom attribute?

Reply:
Oh no, this shouldn't exist, it should be `HTMLAttributes<HTMLDivElement>`

Reply:
I think this one still needs updating.

Reply:
Nevermind, this is the wrong comment.